src/main/java/me/zeroeightsix/kami/module/modules/chat/AntiSpam.kt

```diff
@@ -23,7 +23,7 @@ import java.util.stream.Collectors
  * Updated 12 January 2020 by hub
  * Updated 19 February 2020 by aUniqueUser
  * Updated by dominikaaaa on 19/04/20
- * Updated by Humboldt123 28/05/20 
```

Comment:
This was unintentional

Reply:
It's only removing whitespace, it's fine.